docs/2.guide/2.directory-structure/1.shared.md

```diff
@@ -0,0 +1,98 @@
+﻿---
+title: 'shared'
+head.title: 'shared/'
+description: 'Use the shared/ directory to share functionality between the Vue app and the Nitro server.'
+navigation.icon: 'i-ph-folder'
+---
+
+The `shared/` directory allows you to share code that can be used in both the Vue app and the Nitro server.
+
+::tip
```

Comment:
Personally, I would not put this as a 'tip'; the contents of this body are not optional. How about a `note` instead?

```suggestion
::note
```